share/spack/spackos-bootstrap/stage2/spack.in.yaml

```diff
@@ -0,0 +1,34 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  include:
+    - compilers.yaml
+  specs:
+  - glibc
+  - binutils
+  - gcc@13.2.0 ~bootstrap ~graphite +binutils
+  - gmp
+  # we have no C++ right now, disable
+  - berkeley-db ~cxx ~stl
+  - ncurses ~cxx
```

Comment:
Generally, most specs are missing build deps on `grep` and `sed`, but we can safely assume those are on the host system I guess. `libxcrypt` and `glibc` rely on `/usr/bin/perl` in my case.

Reply:
Ah, yes.  The intention is that stage1 should build everything the other stages need, but it's relying on paths getting set, and I just realized some of this was missing.  I'm working on a fix to keep bootstrap working without relying on host stuff as much, but not explicitly adding all of those yet.